src/ride/track.c

```diff
@@ -4880,15 +4880,162 @@ void game_command_remove_track(int *eax, int *ebx, int *ecx, int *edx, int *esi,
 	return;
 }
 
+uint32 loc_6CDE51() {
+	// mov     ebx, 80000000h
+	return 0x80000000;
+}
+
+uint32 loc_6CDE4C() {
+	// pop     edi
+	// pop     edx
+	// pop     ecx
+	// pop     ebx
+	// pop     eax
+	return loc_6CDE51();
+}
+
+uint32 loc_6CDE43() {
+	// mov     game_command_error_text, 0
+	RCT2_GLOBAL(RCT2_ADDRESS_GAME_COMMAND_ERROR_TEXT, rct_string_id) = 0;
+	
+	return loc_6CDE51();
+}
+
+uint32 loc_6CDE38() {
+	// mov     game_command_error_text, 0
+	RCT2_GLOBAL(RCT2_ADDRESS_GAME_COMMAND_ERROR_TEXT, rct_string_id) = 954; // empty string?
```

Comment:
This probably means that the value is set by some other function rather than loaded from the language file.